[PBLD-48] Change Shape's min size (#449)
Unity-Technologies/com.unity.probuilder: CHANGELOG.md

### Fixed

> - [case: PBLD-48] Fixed a bug where the minimum size for a shape was not taking into account the snap and grid sizes.

Reviewer: ```suggestion
- [case: PBLD-48] Fixed a bug where the minimum size of a shape did not take into account snap and grid sizes.
```
